documentation/quick_start.md

```diff
@@ -58,11 +58,9 @@ You can click the **+** to the right of the tab name just under the main menu to
 
 ## 7. Shut Down Your Server
 
-When you are done with your work session, you must shut down the server you launched.
+Since running sessions continuously use the allocated credit, it strongly recommended that yoy should terminate you session when you are done with the work. See the {ref}`jupyterlab-session-information` for details. The session can be stopped by clicking **Stop My Server** from the jupyterhub control panel page, which is accessed from from:
```

Comment:
It maybe worth chopping up this really long sentence
```suggestion
Since running sessions continuously use the allocated credit, it is strongly recommended that you should terminate your session when you are done with the work. See the {ref}`jupyterlab-session-information` for details. The session can be stopped by clicking **Stop My Server** from the jupyterhub control panel page, which is accessed from from:
```